src/styles/bootstrap-theme/index.scss

```diff
@@ -23,6 +23,5 @@
 @import '../node_modules/bootstrap/scss/variables-dark';
 
 // 4. Include any default map overrides here
-$theme-colors: map-merge($theme-colors, $custom-colors);
```

Comment:
We'll need this line to construct the full theme with custom colors and bootstrap defaults